core-web/libs/sdk/client/src/lib/client/content/methods/get-collection/get-collection.ts

```diff
@@ -0,0 +1,283 @@
+import { Equals } from '../../../../query-builder/lucene-syntax/Equals';
+import { QueryBuilder } from '../../../../query-builder/sdk-query-builder';
+import { CONTENT_API_URL } from '../../shared/const';
+import { GetCollectionResponse, QueryBuilderCallback, SortByArray } from '../../shared/types';
+import { sanitizeQueryForContentType } from '../../shared/utils';
+
+/**
+ * 'GetCollection' Is a Typescript class that provides the ability build a query to get a collection of content.
+ *
+ * @export
+ * @class GetCollection
+ */
+export class GetCollection {
+    private _page = 1;
```

Comment:
@zJaaal Should we consider using ES6 for private attributes? today is compatible with TS and using `#private` attributes these variables are also protected at runtime.

Reply:
I was doing that at first, but wasn't sure about it so I declared them as private. I will change them to `#prop`

Reply:
Done!